azure-resources/ArtificialIntelligence/AI/recommendations.yaml

```diff
@@ -0,0 +1,51 @@
+- description: Ensure to deploy a pay as you go instance of the model when using provisioned throughput to manage overflow effectively
+  aprlGuid: 0c193899-da60-4a52-b4a0-77d75ac8c5c5
+  recommendationTypeId: null
+  recommendationControl: Resiliency
+  recommendationImpact: High
+  recommendationResourceType: Specialized.Workload/AI
+  recommendationMetadataState: Active
+  longDescription: |
+    Provisioned Throughput offers a pre-allocated processing capacity that you pay for regularly, making it suitable for workloads with consistent and predictable usage patterns. In contrast, the Pay-as-You-Go Instance is a flexible billing model where you only pay for the resources you use, ideal for variable or unpredictable workloads. When handling overflow, where demand exceeds the provisioned throughput, the pay-as-you-go instance manages the excess load, ensuring the service remains responsive and efficient during peak times.
+  potentialBenefits: PAYG with provisioned throughput balances cost and performance, enabling scalable AI.
+  pgVerified: true
+  automationAvailable: false
+  tags: null
+  learnMoreLink:
+    - name: Learn More
+      url: "https://learn.microsoft.com/en-us/azure/ai-services/openai/concepts/provisioned-throughput"
+
+- description: Ensure that models are deployed using Global batch for large scale processing
+  aprlGuid: 8aa9744b-f302-4b05-9776-51d6dd3d0c3a
+  recommendationTypeId: null
+  recommendationControl: Scalability
+  recommendationImpact: High
+  recommendationResourceType: Specialized.Workload/AI
```

Comment:
The value needs to be a valid resource type and should align with the directory within which the recommendation is located.